geometry_msgs/JointState joint_state
---
bool success
string message  # for error description